From 1a87f14a2f61993dc76fd7c9bbea074f89e54e22 Mon Sep 17 00:00:00 2001
From: hari gowtham <hgowtham@redhat.com>
Date: Mon, 26 Oct 2015 12:07:16 +0530
Subject: [PATCH 102/131] Remove selinux mount option from "man mount.glusterfs"

Gluster doesn't SELinux mount option, so it is removed.

Change-Id: I471ad98a24b5f5d64279c805b3243cb1168dd3d1
BUG: 1269885
Signed-off-by: hari gowtham <hgowtham@redhat.com>
Reviewed-on: http://review.gluster.org/12422
Tested-by: Gluster Build System <jenkins@build.gluster.com>
Tested-by: NetBSD Build System <jenkins@build.gluster.org>
Reviewed-by: Vijay Bellur <vbellur@redhat.com>
Signed-off-by: Hari Gowtham <hgowtham@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/61349
Reviewed-by: Atin Mukherjee <amukherj@redhat.com>
Tested-by: Atin Mukherjee <amukherj@redhat.com>
---
 doc/mount.glusterfs.8 |    5 +----
 1 files changed, 1 insertions(+), 4 deletions(-)

diff --git a/doc/mount.glusterfs.8 b/doc/mount.glusterfs.8
index 32260ce..8884b48 100644
--- a/doc/mount.glusterfs.8
+++ b/doc/mount.glusterfs.8
@@ -47,9 +47,6 @@ Mount the filesystem with POSIX ACL support
 \fBfopen\-keep\-cache
 Do not purge the cache on file open
 .TP
-\fBselinux
-Enable SELinux label (extened attributes) support on inodes
-.TP
 \fBworm
 Mount the filesystem in 'worm' mode
 .TP
@@ -133,7 +130,7 @@ Use readdirp() mode in fuse kernel module [default: on]
 .I /etc/fstab
 A typical GlusterFS entry in /etc/fstab looks like below
 
-\fBserver1:/mirror  /mnt/mirror  glusterfs log-file=/var/log/mirror.log,acl,selinux   0  0\fR
+\fBserver1:/mirror  /mnt/mirror  glusterfs log-file=/var/log/mirror.log,acl   0  0\fR
 
 .TP
 .I /proc/mounts
-- 
1.7.1

